invenio_rdm_records/cli.py

```diff
@@ -115,22 +119,25 @@ def create_fake_record():
             "publication_date": fake_edtf_level_0(),
             "subjects": [{
                 "subject": fake.word(),
-                "identifier": "subj-1",
-                "scheme": "no-scheme"
+                "identifier": "03yrm5c26",
+                "scheme": "ror"
```

Comment:
later task: I don't know what the subjects vocabulary will be composed of, but it will probably be topical terms. I made a new issue for changing the cli with example subject terms from the vocabularies that InvenioRDM will support by default: https://github.com/inveniosoftware/invenio-rdm-records/issues/337